vscode/src/test/suite/index.ts

```diff
@@ -0,0 +1,40 @@
+import * as path from 'path';
```

Comment:
The codes in this file are borrowed almost from the generated code in https://github.com/microsoft/vscode-generator-code.